### DISCUSSION TITLE
Turbopack freezes, never compile

### DISCUSSION THREAD

**Discussion by user2410** [vercel/turborepo#5644] [Help] [answered]
I've just created a brand new nextjs project. i tried `next dev --turbo` and it stucked at ready state, didn't even throw any error. I made no other configuration. I tried accessing localhost:3000 browser and received infinite loading. 

It works normally without turbo (just `next dev`)

I tried some usual steps: deleting node_modules and .next folder, reinstalling dependencies but nothing works.
https://github.com/vercel/turbo/discussions/2312#discussioncomment-6059224 is the closest description to my problem.

here is my environment info:
- **OS**: Ubuntu 22.04
- **Node**: v19.0.1 
- **yarn**: 1.22.19
- **next**: 13.4.12

steps to reproduce:
1. yarn create next-app@latest my-app --typescript --tailwind --eslint
2. add `--turbo` flag to dev command.

![Screenshot from 2023-07-29 09-08-54](https://github.com/vercel/turbo/assets/89677292/4c1487b7-bad0-4968-aa38-f93a49427725)

**Comment by anthonyshew** [comment, marked as answer]
If you're still seeing this, please open an issue. 🙏
